galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -129,6 +131,15 @@ def validate(self, data):
             )
         return super().validate(data)
 
+    def get_write_only_fields(self, obj):
+        fields = []
+
+        for e in self.Meta.extra_kwargs:
+            if self.Meta.extra_kwargs[e].get('write_only', False):
+                fields.append({"name": e, "is_set": getattr(obj, e) is None})
```

Comment:
`"is_set": getattr(obj, e) is None`

What dows it return if field is not set? like the first time it is requested?